README.md

```diff
@@ -93,13 +128,13 @@ To speed-up a build you can reuse the binaries already downloaded from a previou
 > snapcraft
 ... this build will take a long time and will download all binaries ...
 > cp -r parts/microk8s/build/build/kube_bins .
-> export KUBE_SNAP_BINS=$PWD/kube_bins/v1.10.2/
+> export KUBE_SNAP_BINS=$PWD/kube_bins/v1.10.3/
 > snapcraft
 ... this build will be much faster and will reuse binaries in KUBE_SNAP_BINS
 
 ```
 
 ### Installing the snap
 ```
-snap install microk8s_v1.10.3_amd64.snap --classic --dangerous
+sudo snap install microk8s_v1.10.3_amd64.snap --classic --dangerous
```

Comment:
I'd prefer not to use sudo for these commands, snap will tell you to use sudo if you have to (snap login will prevent sudo)